AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -913,7 +924,7 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertSectionsAt
 - (void)rangeController:(ASRangeController *)rangeController didDeleteSectionsAtIndexSet:(NSIndexSet *)indexSet withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions
 {
   ASDisplayNodeAssertMainThread();
-  
+  [_layoutFacilitator collectionViewEditingSectionsAtIndexSet:indexSet];
```

Comment:
As far as I can tell, full reloads and move operations are both done through deleting and inserting nodes, and these delegates will be called.